_layouts/books.html

```diff
@@ -6,7 +6,54 @@
       {%- include waves.html -%}
     </header>
     <div class="book-wrapper">
-        {{ content }}
+      {% assign pageCount=0 %}
+      {% assign year = page.year | append: ""%}
+      {% assign prevYear = page.year | minus: '1' | append: ""%}
+      {% assign nextYear = page.year | plus: '1'  | append: ""%}
+      <div class="content">
+        <div class="flex-wrapper">
+          <div class="log-header">
+            {% if site.data.reading[prevYear] %}
```

Comment:
If I ever make a new data file for a year but forget to make that year's .md file then this will create a link to a nonexisting page. Maybe not ideal but seemed easy enough to manage.